lab_3_generate_by_ngrams/main.py

```diff
@@ -159,6 +235,13 @@ def _postprocess_decoded_text(self, decoded_corpus: tuple[str, ...]) -> Optional
 
         In case of corrupt input arguments, None is returned
         """
+        if not isinstance(decoded_corpus, tuple) or\
+                len(decoded_corpus) == 0:
+            return None
+        text_string = ''.join(list(decoded_corpus))
+        text = ' '.join(text_string.split('_'))
+        text = text.capitalize().rstrip() + '.'
```

Comment:
Is it possible that the `text` will start with space? In this case, `rstrip` won't work...

Reply:
Fixed